src/index.spec.js

```diff
@@ -273,6 +273,29 @@ const tests = [
     expected:
       "For I killed a man for my wound,\neven a young man for my bruise",
   },
+  {
+    params: {
+      name: "",
+      bookId: "1PE",
+      ref: "1:24",
+      quote:
+        "πᾶσα σὰρξ ὡς χόρτος, καὶ πᾶσα δόξα αὐτῆς ὡς ἄνθος χόρτου. ἐξηράνθη ὁ χόρτος, καὶ τὸ ἄνθος ἐξέπεσεν,",
```

Comment:
Fails because of the period. If no period, or a new line is after the period, it will pass.